doc/tests/mapping.rst

```diff
@@ -0,0 +1,13 @@
+``mapping``
+============
+
+``mapping`` checks if a variable is a mapping:
+
+.. code-block:: twig
+
+    {# evaluates to true if the users variable is a mapping #}
+    {% if users is mapping %}
+        {% for key, user in users %}
+            {{ key }}: {{ user }};
+        {% endfor %}
+    {% endif %}
```

Comment:
Maybe we can a note about how we define a mapping?

Reply:
sure

Reply:
Done in https://github.com/twigphp/Twig/pull/3859/commits/9ecaf164fcf9de0a2f782153dbfa235d88a4efb6